pkg/kv/kvserver/client_raft_test.go

```diff
@@ -3002,11 +3002,11 @@ func TestRemovePlaceholderRace(t *testing.T) {
 			for {
 				chgs := roachpb.MakeReplicationChanges(action, tc.Target(1))
 				if _, err := repl.ChangeReplicas(ctx, repl.Desc(), kvserverpb.SnapshotRequest_REBALANCE, kvserverpb.ReasonUnknown, "", chgs); err != nil {
-					if kvserver.IsRetriableReplicationChangeError(err) {
+					if kvserver.IsRetriableReplicationChangeError(err) ||
+						kvserver.IsReplicationChangeInProgressError(err) {
```

Comment:
I think that could make sense. The only non-test caller is `executeAdminCommandWithDescriptor`, which I think would want to retry replication change errors as well. However, let's submit a follow-up PR for that, so that this is trivially backportable.